README.md

```diff
@@ -1,18 +1,16 @@
-# nuxt-device-detect
+# @nuxtjs/device
 
 This module injects flags that indicate a device type into the context and the component instance.
 
-[![nuxt-device-detect Dev Token](https://badge.devtoken.rocks/nuxt-device-detect)](https://devtoken.rocks/package/nuxt-device-detect)
```

Comment:
I think you can still keep this badge :) (Maybe plus standard badges of the community for NPM, CI, Coverage, ...)

Reply:
Not mandatory for this PR just a comment.